agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -594,6 +595,37 @@ void backhaul_manager::after_select(bool timeout)
         discovery_timestamp = now;
         send_1905_topology_discovery_message();
     }
+
+    // Each platform must send Topology Discovery message every 60 seconds to its first circle
+    // 1905.1 neighbors.
+    // Iterate on each of known 1905.1 neighbors and check if we have received in the last
+    // 60 seconds a Topology Discovery message from it. If not, remove this neighbor from our list
+    // and send a Topology Notification message.
+    bool neighbors_list_changed = false;
+    for (auto it = m_1905_neighbor_devices.begin(); it != m_1905_neighbor_devices.end();) {
+        const auto &last_topology_discovery = it->second;
+        if (last_topology_discovery + std::chrono::seconds(DISCOVERY_NEIGHBOUR_REMOVAL_TIMEOUT) <
+            std::chrono::steady_clock::now()) {
+            const auto &device_al_mac = it->first;
```

Comment:
In this case I think it is good to create the variable since it tells the reader that `it->first` is the device al mac, and will be optimized anyway by the compiler.

Reply:
Yes, that what I've tried to do.